src/timepicker/timepicker.spec.ts

```diff
@@ -21,12 +21,15 @@ function getMeridianButton(nativeEl: HTMLElement) {
 function expectToDisplayTime(el: HTMLElement, time: string) {
   const inputs = getInputs(el);
   const timeParts = time.split(':');
+  let timeInInputs = [];
 
   expect(inputs.length).toBe(timeParts.length);
 
   for (let i = 0; i < inputs.length; i++) {
-    expect((<HTMLInputElement>inputs[i]).value).toBe(timeParts[i]);
+    timeInInputs.push((<HTMLInputElement>inputs[i]).value);
```

Comment:
Yeh, wanted to do it but the list we get from DOM query doesn't have `map` method so we would have to convert it to an array and it is not as nice any more.

Going to keep it as-is for now but if you can see how to simplify it please send a PR